src/node/app.ts

```diff
@@ -11,7 +11,7 @@ import { disposer } from "./http"
 import { isNodeJSErrnoException } from "./util"
 import { handleUpgrade } from "./wsRouter"
 
-type ListenOptions = Pick<DefaultedArgs, "socket" | "port" | "host">
+type ListenOptions = Pick<DefaultedArgs, "socket-permission" | "socket" | "port" | "host">
```

Comment:
I do not feel strongly either way but what do we think about `--socket-mode` to match things like systemd?

Reply:
have changed to --socket-mode